src/components/Link.react.js

```diff
@@ -2,69 +2,72 @@
 
 import PropTypes from 'prop-types';
 
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 
 /*
  * event polyfill for IE
  * https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent
  */
-function CustomEvent (event, params) {
-    params = params || {
-        bubbles: false,
-        cancelable: false,
-        detail: undefined
-    };
-    var evt = document.createEvent('CustomEvent');
-    evt.initCustomEvent(
-        event, params.bubbles, params.cancelable, params.detail);
-    return evt;
+function CustomEvent(event, params) {
+  params = params || {
+    bubbles: false,
+    cancelable: false,
+    detail: undefined
+  };
+  var evt = document.createEvent('CustomEvent');
+  evt.initCustomEvent(event, params.bubbles, params.cancelable, params.detail);
+  return evt;
```

Comment:
In general, it'd be helpful if you split out commits that did style updates vs commits that added or removed code. As a reviewer, it takes more time to understand which lines have actual changes

Reply:
I think this is another case of the Prettier autoformatter doing "it's job" :) But duly noted!